tools/c7n_org/c7n_org/cli.py

```diff
@@ -405,13 +405,13 @@ def _get_env_creds(session, region):
 
 
 def run_account_script(account, region, output_dir, debug, script_args):
-    try:
+    env = os.environ.copy()
+
+    if account.get('role') or account.get('profile'):
```

Comment:
afaicr, we still want the try / catch on get_session here, people often misconfigure roles in aws

so the notion here is for gcp and azure is that the principal already has multi project or multi subscription access, and we don't need a session? don't we need to explicitly populate the provider specific standard sdk env credentials to execute on the right gcp project?